.gitignore

```diff
@@ -124,3 +124,38 @@ env/
 ### Configuration files ###
 .env
 .env.json
+
+# Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+
+# Crash log files
+crash.log
+crash.*.log
+
+# Exclude all .tfvars files, which are likely to contain sensitive data, such as
+# password, private keys, and other secrets. These should not be part of version 
+# control as they are data points which are potentially sensitive and subject 
+# to change depending on the environment.
+*.tfvars
+*.tfvars.json
+
+# Ignore override files as they are usually used to override resources locally and so
+# are not checked in
+override.tf
```

Comment:
Possibly something you're already conscious of, but one I like to advertise whenever we copy changes between `.dockerignore` and `.gitignore` is [that they use slightly different syntaxes that can catch you out](https://zzz.buzz/2018/05/23/differences-of-rules-between-gitignore-and-dockerignore/) - I can't see any issues with these as is but you are excluding slightly different patterns (e.g. all files called `override.tf` are git-ignored, but only ones at the root level are docker-ignored)